classes/Smarty/SmartyResourceModule.php

```diff
@@ -31,10 +31,14 @@
  */
 class SmartyResourceModuleCore extends Smarty_Resource_Custom
 {
-    public function __construct(array $paths, $isAdmin = false)
+    /**
+     * @var array<string>
+     */
+    public $paths;
+
+    public function __construct(array $paths)
```

Comment:
BC breaks :thinking: 
Why removing the isAdmin?

Reply:
Why use
```php
$smarty->registerResource('module', new SmartyResourceModule($module_resources, $isAdmin = true));
```

Reply:
It's a public property, it's maybe used by modules

Reply:
I re-add it.